myojana/myojana/report/total_approved_demands/total_approved_demands.py

```diff
@@ -17,9 +17,9 @@ def execute(filters=None):
         }
     ]
 
-    condition_str = ReportFilter.set_report_filters(filters, 'date_of_visit', True)
+    condition_str = ReportFilter.set_report_filters(filters, 'creation', True)
     if condition_str:
-        condition_str = f"AND {condition_str}"
+        condition_str = f"AND {condition_str} AND 1=1"
```

Comment:
Relook into this code and test it.

Reply:
Hi @amreshyadavgit ,
Thanks for you suggestion .
As per your suggestion I have change the code and test please look into it and verify if possible